lib/nebulex/adapters/multilevel.ex

```diff
@@ -411,6 +411,11 @@ defmodule Nebulex.Adapters.Multilevel do
     |> Enum.reduce(object, &replicate(&1, &2))
   end
 
+  defp replicate(cache, %Object{expire_at: :infinity} = object) do
```

Comment:
I think patter-match should be against `nil` instead of `:infinity` (according the `Nebulex.Object` struct). Otherwise, we should check why `:infinity` is being set in the object.

Reply:
Well...`nil` would be handled by the original function. As far as where `:infinity` is coming from, thats a good question.

Reply:
I would suggest doing a search in the repo for `:infinity` because there seem to be some things which use `nil` and some which use `:infinity`.

Reply:
Yeah, meanwhile, let's merge this PR, it fixes the issue clearly, then I will find where the `:infinity` is coming from. Thanks for your contribution :) !!